tests/webview/visual/test_home_objects.py

```diff
@@ -0,0 +1,36 @@
+from tests import markers
+from pages.webview.home import Home
+
+APP_NAME = 'CNX HOMEPAGE BOOKS VISUAL VERIFICATION-TEST1'
+
+@markers.webview
+@markers.nondestructive
+def test_home_objects(applitools, webview_base_url, selenium):
+
+    try:
+
+        home = Home(selenium, webview_base_url).open()
+
+        # Start the test and set the browser's viewport size to 1400x820
+        applitools.open(driver=selenium, app_name=APP_NAME, test_name='Test homepage books', viewport_size={'width': 1400, 'height': 820})
```

Comment:
We should try to avoid magic numbers. We may want to pull this off of the home page. something like this:

`applitools.open(driver=selenium, app_name=APP_NAME, test_name='Test homepage books', viewport_size={'width': home.window_width, 'height': home.window_height})`

Reply:
if home.window_width doesn't exist it will need to be implemented.

Reply:
I am using this logic adopted from test_home.py:
```
@markers.parametrize("width,height", [(1400, 820)])
def test_home_objects(applitools, webview_base_url, selenium, width, height):
...
    try:
        home = Home(selenium, webview_base_url).open()

        # Start the test and set the browser's viewport size to 1400x820
        applitools.open(driver=selenium, app_name=APP_NAME, test_name='Test homepage books', viewport_size={'width': width, 'height': height})
```
This seems to be working and takes away magic numbers.